### PR TITLE
docs(specs): S5 child spec — spec contract (grill admissions, AC4 spec lint, brainstorming teardown)

### PR DESCRIPTION
**Pull request by scotthamilton77**
## Summary

Child spec for charter slice S5 (`docs/specs/2026-07-21-harness-rework-way-forward.md`, D1/D2/D18, AC4 + the writing-plans clause of AC5), following the S2 child-spec pattern (inventory → decisions → per-slice red-test-convertible ACs → out of scope).

- **Slice A** — admit `grill-me` (promoted from oss-snapshots) and regularize `grill-with-docs`, each with an exit-criterion graft (enumerated AC IDs + edge-case taxonomy) and the first real `admission:` records; adapt `to-spec` with an AC-section + slice-list output contract; drift policy flips to local-fork.
- **Slice B** — AC4 spec lint as `packages/installer/core/spec_lint.py` beside the AC1/AC3 enforcement, date-gated ≥ 2026-07-24 (self-hosting: this spec is inside the gate).
- **Slice C** — delete `brainstorming` (goals-only escape) and `writing-plans`, re-point the 9-file blast radius, leave generic-word prose mentions alone.

Tracker: agents-config-9k9.15 (spec task agents-config-9k9.15.1.1).

Review focus per D3's spirit: attack the AC set — name behaviors that satisfy these ACs while still being wrong.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01UJ7AFo9nW6JTJ85QEbCtzp

### CODE REVIEW

**Comment by scotthamilton77** [comment]
@codex review

File: docs/specs/2026-07-24-spec-contract-s5.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+**S5-D3 — `to-spec` is adapted, not copied.** Imported to
+`src/user/.agents/skills/to-spec/` with its output contract replaced: the
+embedded template gains an **Acceptance Criteria** section (IDs,
+red-test-convertible, taxonomy applied) and an **ordered slice list** with
+per-slice AC citations and the D2 size tripwire (spec > 400 lines or > 8
+slices splits). The upstream "publish to tracker with `ready-for-agent`
+label" step is dropped — output lands as a dated file in the project's spec
+home; tracker publication is `to-tickets`' seat (admitted separately per
+D18). `disable-model-invocation: true` is retained (user-invoked).
```
[the remark this answers is not in the document]
Comment (scotthamilton77) [reply docs/specs/2026-07-24-spec-contract-s5.md]: Accepted (commit 329de9e): the shared-tree to-spec source drops `disable-model-invocation` — user-invoked-only intent is now carried portably in the description, and any Claude-side invocation hard-stop would be a separately admitted Claude-tree override. S5-D3 and S5-A3 updated.

File: docs/specs/2026-07-24-spec-contract-s5.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+**S5-D5 — The spec lint lives with the other structural-AC enforcement.**
+`packages/installer` already enforces AC1 (`surface_budget.py`) and AC3
+(`admission.py`); AC4 lands beside them as `core/spec_lint.py` with a CLI
+entry, a `make spec-lint` target, and a step in the CI gate. Scope: files
+matching `docs/specs/YYYY-MM-DD-*.md` with date ≥ **2026-07-24** (legacy
+specs are exempt by date, no allowlist file; this spec itself is inside the
+gate — self-hosting). Mechanical checks only: (1) an `Acceptance criteria`
+heading exists (case-insensitive); (2) at least one AC ID matching
+`[A-Z0-9]+-[A-Z]\d+|AC\d+` is defined under it; (3) if slice headings exist
+(`Slice` in a heading), every slice section cites ≥ 1 AC ID. Prose quality
```
[the remark this answers is not in the document]
Comment (scotthamilton77) [reply docs/specs/2026-07-24-spec-contract-s5.md]: Accepted (commit 329de9e): the lint now requires structured AC definition entries (`- **<ID>** <text>`, non-empty text) under the heading, extracts the defined-ID set, and requires each slice to cite an ID from that set — bare tokens define nothing and undefined-ID citations fail. S5-D5, S5-B2, S5-B3 updated with both gaming cases as explicit red fixtures.
